requirements/updater.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/sh
 set -ue
 
-requirements_in="$(readlink -f ./requirements.in)"
-requirements="$(readlink -f ./requirements.txt)"
-requirements_git="$(readlink -f ./requirements_git.txt)"
+if [[ $OSTYPE == 'darwin'* ]]; then
+	requirements_in="$(python -c 'import os,sys;print(os.path.realpath(sys.argv[1]))' ./requirements.in)"
+	requirements="$(python -c 'import os,sys;print(os.path.realpath(sys.argv[1]))' ./requirements.txt)"
+	requirements_git="$(python -c 'import os,sys;print(os.path.realpath(sys.argv[1]))' ./requirements_git.txt)"
```

Comment:
why don't we use this syntax in both cases? I don't see any reason this wouldn't work on Linux.